Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -955,13 +976,21 @@
     return false;
 }
 
-bool UnifiedPDFPlugin::handleEditingCommand(StringView commandName)
+bool UnifiedPDFPlugin::handleEditingCommand(const String& commandName, const String& argument)
 {
+    if (!m_frame || !m_frame->coreLocalFrame())
+        return false;
+    if (commandName == "ScrollPageBackward"_s || commandName == "ScrollPageForward"_s) {
+        m_frame->coreLocalFrame()->checkedEditor()->command(commandName).execute(argument);
+        return true;
+    }
     return false;
```

Comment:
Nit:
```suggestion
    if (isEditingCommandEnabled(commandName)) {
        m_frame->coreLocalFrame()->checkedEditor()->command(commandName).execute(argument);
        return true;
    }
    return false;
```

Reply:
This is definitely wrong. Technically correct at the moment, semantically completely wrong.

Reply:
Oh! My bad... it read nicely in my head. 🤣

Reply:
If you peek at PDFPlugin's impl you'll see that the others won't forward, they'll do specific things instead! That's just the implementation for these two.